.github/workflows/reusable-publish.yml

```diff
@@ -39,9 +39,9 @@ jobs:
           python-version: 3.x
       - name: Generate release notes
         id: release_notes
-        run: ./.github/scripts/generate_release_notes.sh $PACKAGE
+        run: ./.github/scripts/generate_release_notes.sh ${{ inputs.package }}
         env:
-          PACKAGE: ${{ inputs.package }}
+          GH_TOKEN: ${{ github.token }}
```

Comment:
What is the difference between

`GH_TOKEN: ${{ github.token }}`
and
`GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` ?

Reply:
${{ github.token }} -> the default token
${{ secrets.GITHUB_TOKEN }} -> stored in the secrets. usually a PAT with higher priviledges